music_perception/Piece piece

string finger # one of th/ff/mf/rf/lf
---
---
